pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -68,32 +69,32 @@ Refer to [Mount Volume Docs](/1.13/storage/mount-disk-resources/) to create moun
 
 ### S3 Storage Option
 
-Given the default configuration of the DC/OS Package Registry, DC/OS Packages are stored in a local persistent volume in the host filesystem. When using this default storage configuration, you are limited to one instance of the registry. Package Registry also supports a highly available configuration by storing DC/OS Packages on a S3 compatible storage which supports deploying more than one instance of the registry.
+Given the default configuration of the DC/OS Package Registry, DC/OS Packages are stored in a local persistent volume in the host filesystem. When using this default storage configuration, you are limited to one instance of the registry. Package Registry also supports a highly available configuration by storing DC/OS Packages on S3-compatible storage, which supports deploying more than one instance of the registry.
```

Comment:
Replace `filesystem` with `file system`.

Reply:
Thanks for catching that!